src/stem.js

```diff
@@ -84,8 +88,9 @@ export class Stem extends Element {
 
   // Gets the entire height for the stem
   getHeight() {
+    const y_offset = (this.stem_direction === Stem.UP) ? this.stem_up_y_offset : this.stem_down_y_offset; // eslint-disable-line max-len
```

Comment:
this could be `const y_offset = this.y_offset * (this.stem_direction === Stem.UP ? 1 : -1)`

Reply:
Agreed, when we have a single offset it could be